.github/CODEOWNERS

```diff
@@ -363,23 +363,23 @@
 /core/controllers/skill*.py @aks681
 /core/domain/skill*.py @aks681
 /core/storage/skill/ @aks681
-/core/templates/components/entity-creation-services/skill-creation.service.ts @kevintab95
+/core/templates/components/entity-creation-services/skill-creation.service.ts @kevintab95 @Showtim3
```

Comment:
If the point is to distribute things, you should probably remove @kevintab95 (here and below)?

Reply:
My 2c: If there are going to be large changes to editor pages, and if you would like me to take a look, then let's keep the editor pages assigned to both of us until major changes are done. If not, I'm fine with the change @seanlip suggested.

Reply:
FYI I think all major changes are done for: topics/skills dashboard; topic editor; skill editor; story editor; chapter editor.

Reply:
Ah ok. Sounds good, in that case.

Reply:
I'd say lets both be assigned for a while and when we think that I am sufficient to review alone(dont want to miss out some bugs while reviewing and let them pass to the codebase), then we'll remove Kevin's assignment. @kevintab95 @seanlip Hope that's fine?

Reply:
Actually, for that, what you should do is: still review alone, but loop Kevin in after you've done a review (send him a ping) and ask him to take a pass and see if he spots anything you missed. Would that be fine?

Reply:
That sounds great. Done.